.ui/pgCategories.pgx

```diff
@@ -87,18 +67,20 @@
 				"layoutManager": {
 					"spanCount": 2
```

Comment:
we shouldn't be using user props inside page

Reply:
done.